{
  "metadata": {
    "timestamp": "2025-02-26T16:13:47Z"
  },
  "data": [
    {
      "cve": {
        "id": "CVE-2019-13926",
        "description": "A vulnerability has been identified in SCALANCE S602 (All versions \u003e= V3.0 and \u003c V4.1), SCALANCE S612 (All versions \u003e= V3.0 and \u003c V4.1), SCALANCE S623 (All versions \u003e= V3.0 and \u003c V4.1), SCALANCE S627-2M (All versions \u003e= V3.0 and \u003c V4.1). Specially crafted packets sent to port 443/tcp of affected devices could cause a Denial-of-Service condition of the web server. A cold reboot is required to restore the functionality of the device.",
        "metadata": {
          "published": "",
          "last_modified": "",
          "confidence_level": "low",
          "severity": "high",
          "source": "",
          "status": ""
        },
        "impact": {
          "cisa_kev": false,
          "reported_exploited": false,
          "exploit_maturity": "none",
          "automatable": true
        },
        "metrics": [
          {
            "source": "nvd@nist.gov",
            "type": "Primary",
            "cvss_version": "3.1",
            "vector_string": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
            "attack_vector": "NETWORK",
            "base_score": 7.5
          },
          {
            "source": "nvd@nist.gov",
            "type": "Primary",
            "cvss_version": "2.0",
            "vector_string": "AV:N/AC:L/Au:N/C:N/I:N/A:C",
            "attack_vector": "NETWORK",
            "base_score": 7.80000019073486
          }
        ],
        "timeline": {
          "references": null,
          "repositories": null
        },
        "epss": {
          "score": 0.00103,
          "percentile": 0.43678
        },
        "counts": {
          "public_exploit_count": 0
        }
      }
    }
  ]
}